velox/exec/SimpleAggregateAdapter.h

```diff
@@ -44,6 +44,21 @@ class SimpleAggregateAdapter : public Aggregate {
   explicit SimpleAggregateAdapter(TypePtr resultType)
       : Aggregate(std::move(resultType)) {}
 
+  // Function-level states are variables hold by a UDAF instance that are
+  // computed once and used at every row when adding inputs to accumulators or
+  // extracting values from accumulators.
+  typename FUNC::FunctionState state_;
+
+  void initialize(
+      core::AggregationNode::Step step,
+      const std::vector<TypePtr>& rawInputTypes,
+      const TypePtr& resultType,
+      const std::vector<VectorPtr>& constantInputs) override {
+    if constexpr (support_initialize_function_state_) {
+      FUNC::initialize(state_, rawInputTypes, resultType, constantInputs);
```

Comment:
Let's also pass `step` to FUNC::initialize() in case it's useful for the general use case.